pkg/limayaml/validate.go

```diff
@@ -277,6 +277,7 @@ func Validate(y *LimaYAML, warn bool) error {
 			if _, err := os.Stat(playbook); err != nil {
 				return fmt.Errorf("field `provision[%d].playbook` refers to an inaccessible path: %q: %w", i, playbook, err)
 			}
+			logrus.Warnf("ansible provision mode is deprecated, use `ansible-playbook %q` instead", playbook)
```

Comment:
I'm not sure it is clear to the user that `ansible-playbook` should be run directly (on the host? inside the VM?), and that it isn't a new `lima.yaml` setting.

Nit-picking: `provision mode "ansible"`